recipes/pyspark/pysparkretailapp/helper.py

```diff
@@ -90,5 +94,5 @@ def prepare_dataset(config_properties, dataset):
     return pd
 
 def get_dataset_options(spark_context):
-    dataset_options = spark_context._jvm.com.adobe.platform.dataset.DataSetOptions
+    dataset_options = spark_context._jvm.com.adobe.platform.query.QSOption
```

Comment:
Could we rename this to `get_query_options` or something so it better matches the updated names?  Then change all references of `dataset_options` to `query_options` as well?